ui/partials/perf/revisiondescribe.html

```diff
@@ -0,0 +1,14 @@
+<li>
+    <strong>Base</strong> - <a href="{{originalProject.getRevisionHref(originalRevision)}}">{{originalRevision}}</a> ({{originalProject.name}}) - {{originalResultSet.author}} -
+    <span ng-show="originalResultSet.comments.length < 60">{{originalResultSet.comments}}</span>
+    <span ng-show="originalResultSet.comments.length > 60">{{originalResultSet.comments.substring(0, 60)}}
```

Comment:
You can simplify lines 3 and 4 to:

    <span>{{originalResultSet.comments.substring(0, 60)}}</span>

If the second parameter in substring is greater than the length of the string, it is automatically truncated to the length of the string:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring